src/logic/tokens/store/actions/fetchSafeTokens.ts

```diff
@@ -26,7 +35,7 @@ const fetchSafeTokens = (safeAddress) => async (dispatch, getState) => {
       return
     }
 
-    const result = await fetchTokenCurrenciesBalances(safeAddress)
+    const result = await backOff(() => fetchTokenCurrenciesBalances(safeAddress))
```

Comment:
let's use a native axios plugin instead of this:
https://www.npmjs.com/package/axios-retry
or
https://www.npmjs.com/package/retry-axios

Reply:
If we use that we won't be able to implement backoff for methods like this: `const newEthBalance = await backOff(() => getBalanceInEtherOf(safeAddress))` because they use web3 methods

Reply:
Valid point